sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/Readme.md

```diff
@@ -68,7 +68,7 @@ Use `ModelSettings` in [TimeSeriesInsightsClient](https://github.com/Azure/azure
 
 ```C# Snippet:TimeSeriesInsightsSampleGetModelSettings
 Response<TimeSeriesModelSettings> getModelSettingsResponse = await client.ModelSettings.GetAsync();
-Console.WriteLine($"Retrieved Time Series Insights model settings:\n{JsonSerializer.Serialize(getModelSettingsResponse.Value)}");
+Console.WriteLine($"Retrieved Time Series Insights model settings \nname : '{getModelSettingsResponse.Value.Name}', default type Id: {getModelSettingsResponse.Value.DefaultTypeId}, '{JsonSerializer.Serialize(getModelSettingsResponse.Value.TimeSeriesIdProperties)}'");
```

Comment:
I would break this line into multiple lines. I will make the change on your branch and push the change.

Reply:
I would actually go as far as looping through the TimeSeriesIdProperties and print the name and type for each id property. Thoughts?